### DISCUSSION TITLE
Plan to support FCM?

### DISCUSSION THREAD

**Discussion by richard457** [invertase/flutterfire_desktop#17] [Ideas]
Notification is essential for any app, it could be nice to have FCM implemented too, any plans?

**Comment by pr-Mais** [comment]
The work is currently focused on Authentication, there's no plan decided but certainly there're other packages to be supported in the future.

Regarding FCM, refer to this answer: https://github.com/invertase/flutterfire_desktop/discussions/10#discussioncomment-1563866
